### PR TITLE
Fix review comments

### PR DESCRIPTION
**Pull request by mkumatag**
This PR will fix following review comments:
- Global yes/no option for unattended installation
- Add apt-get update before any install
- Remove -D option from docker-machine command
- Print additional banner to relogin after the installation

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -5,11 +5,24 @@
 # This project is licensed under the Apache License 2.0, see LICENSE.
 #
 # Install script for installing boot2docker for power architecture
+# Usage:
+#	./install [OPTION]
+#	Options:
+#		yes/no   - Answer for all the question, pass yes for unattended installation
```
Comment (bpradipt) [inline install.sh]: yes/no seems confusing. Why not we just use an option like silent (-s) for unattended install. The default will be attended ?

Comment (mkumatag) [inline install.sh]: okay..

Comment (mkumatag) [inline install.sh]: @bpradipt addressed the comments, PTAL at latest commit.

File: install.sh [hunk 2]
```diff
@@ -5,11 +5,24 @@
...
 ##################################################################
 
 #set -x
 
+silent=${1:-}
+
+if [[ -n $silent && ( "$silent" != "-s" && "$silent" != "--silent") ]]; then
+	echo "Invalid option : $@"
+	echo "Usage:"
+	echo -e "\t./install yes/no"
```
Comment (bpradipt) [inline install.sh]: The message needs modification. There is no yes/no option anymore

File: install.sh [hunk 3]
```diff
@@ -5,11 +5,24 @@
...
+	exit 1
+fi
+
 VERSION="17.03.0-ce"
 ISOPATH="$HOME/.docker/machine"
 ISOFILE="boot2docker-${VERSION}.iso"
```
Comment (bpradipt) [inline install.sh]: Remove the ToDo comment. No longer needed

**Pull request merged**